tmux reports as version 3.4, as it's using an unreleased commit version to work
around issues with ncurses.

diff --git a/lib/tmuxinator/tmux_version.rb b/lib/tmuxinator/tmux_version.rb
index 5a0a6b0..0a88381 100644
--- a/lib/tmuxinator/tmux_version.rb
+++ b/lib/tmuxinator/tmux_version.rb
@@ -1,6 +1,7 @@
 module Tmuxinator
   module TmuxVersion
     SUPPORTED_TMUX_VERSIONS = [
+      3.4,
       "3.3a",
       3.3,
       "3.2a",
